.github/component_owners.yml

```diff
@@ -6,6 +6,10 @@ components:
     - carolabadeer
   detectors/node/opentelemetry-resource-detector-container:
     - abhee11
+  detectors/node/opentelemetry-resource-detector-gcp: []
+    # Unmaintained?
+  detectors/node/opentelemetry-resource-detector-github: []
```

Comment:
The original contributor here was @smithclay. Maybe they would like to take ownership?